messagebus/src/main/java/com/yahoo/messagebus/routing/RoutingNode.java

```diff
@@ -325,11 +329,14 @@ private boolean hasUnconsumedErrors() {
                         it = it.parent;
                     }
                     if (errorCode != ErrorCode.NONE) {
+                        if (errors == null) {
+                            errors = new StringBuilder();
+                        }
+                        errors.append(error.toString());
```

Comment:
Should we have a delimiter between multiple, individual errors, or are they already in a "self-delimiting" format?

Reply:
Ignore this, looked at single commit by itself :)